### ISSUE TITLE
Error when running the tool

### ISSUE THREAD

**Issue by rafikramsis** [ErikEJ/EFCorePowerTools#1251]
### Steps to reproduce

I run the tool, after I select database connection, selected EF 5, I get the below error in output window in VS:

System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at bool System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at Process System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at async Task<string> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.RunProcessAsync(ProcessStartInfo startInfo)
   at async Task<bool> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.IsDotnetInstalledAsync(string version)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesInternalAsync(string arguments)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesAsync(string connectionString, DatabaseType databaseType, SchemaInfo[] schemas, bool mergeDacpacs)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.TableListBuilder.GetTableDefinitionsAsync(CodeGenerationMode codeGenerationMode)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.GetTablesAsync(DatabaseConnectionModel dbInfo, CodeGenerationMode codeGenerationMode, SchemaInfo[] schemas)
   at async Task<bool> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.LoadDataBaseObjectsAsync(ReverseEngineerOptions options, DatabaseConnectionModel dbInfo, Tuple<List<Schema>, string> namingOptionsAndPath)
   at async Task EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.ReverseEngineerCodeFirstAsync(Project project, string optionsPath, bool onlyGenerate)

### Further technical details

EF Core version in use: EF Core 5.0

EF Core Power Tools version:  2.5.888

Database engine: SQL Server

Visual Studio version:  Visual Studio 2022 17.05

**Comment by ErikEJ** [comment]
Maybe I did a breaking change in the latest daily? rying to revert now.

Otherwise try to remove the %temp%\efpt*.* folder(s)

**Comment by rafikramsis** [comment]
Hi Erik, 

Thanks for the fast reply, I cleared the efpt folder under %temp% and downloaded the latest build of the tool (2.5889) but still get's the below error in VS output:

System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at bool System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at Process System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at async Task<string> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.RunProcessAsync(ProcessStartInfo startInfo)
   at async Task<bool> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.IsDotnetInstalledAsync(string version)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesInternalAsync(string arguments)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesAsync(string connectionString, DatabaseType databaseType, SchemaInfo[] schemas, bool mergeDacpacs)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.TableListBuilder.GetTableDefinitionsAsync(CodeGenerationMode codeGenerationMode)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.GetTablesAsync(DatabaseConnectionModel dbInfo, CodeGenerationMode codeGenerationMode, SchemaInfo[] schemas)
   at async Task<bool> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.LoadDataBaseObjectsAsync(ReverseEngineerOptions options, DatabaseConnectionModel dbInfo, Tuple<List<Schema>, string> namingOptionsAndPath)
   at async Task EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.ReverseEngineerCodeFirstAsync(Project project, string optionsPath, bool onlyGenerate)


I tried running VS as admin, didn't help!

I tried as well to uninstall the tool and installed this version: EF_Core_Power_Tools_v2.5.877, same error!

Any ideas what else I can try?
